source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -174,7 +174,7 @@ class CalendarExceptionSerializer(PbNestedSerializer):
 
 
 class CalendarSerializer(PbNestedSerializer):
-    id = jsonschema.MethodField(schema_type=str, display_none=True, description='Identifier of the object')
+    id = jsonschema.MethodField(schema_type=str, display_none=False, description='Identifier of the object')
```

Comment:
we may have to check what marshal does but it may be a bit dangerous for integrators, no?
(well integrators that use calendars 😄 )
If tests were breaking because of that "True" then it's OK, integrators were at risk before (well... if they ever used /v_j).